main.py

```diff
@@ -100,6 +102,7 @@ def update_colors(self, links):
                                 self.switches[neighbor]['color'],
                                 self._color_field
                             )
+                        self.set_flow_table_group_owner(flow_dict, "base")
```

Comment:
Since `"base" is optional in the arg, no need to pass here, unless it changes down in the future, but if ever does, we'll take care in the future.

Reply:
Deleted.